src/LiveComponent/doc/index.rst

```diff
@@ -3715,6 +3715,11 @@ Debugging Components
 Need to list or debug some component issues.
 The `Twig Component debug command`_ can help you.
 
+To debug your PHP code, you can inject a `private LoggerInterface $logger` in your `__construct`.
+Then you can use `$this->logger->debug('Message', ['var' => $var]);` and see the output in `var/log/dev.log`.
```

Comment:
I'm not sure this is really needed, as it applies to any component / service in Symfony apps (and you can inject a Logger with a lot of other methods,) . 

We could send the user towards the documentation pages on logging, but then it would not be really about _debugging_.